lib/chef/knife/clc_server_create.rb

```diff
@@ -520,6 +546,44 @@ def sync_bootstrap(uuid)
         retry_on_timeouts { command.run }
       end
 
+      def sync_win_bootstrap(uuid)
+        server = connection.show_server(uuid, true)
+        ensure_server_powered_on(server)
+        creds = get_server_credentials(server)
+        command = bootstrap_windows_command
+        command.config[:winrm_user] = creds["userName"]
+        command.config[:winrm_password] = creds["password"]
+        command.config[:winrm_transport] = :sspinegotiate if os_windows?
```

Comment:
Why do you override it here? What if user would like to use `kerberos`? Or just plain `basic` by himself?